cgsmiles/write_cgsmiles.py

```diff
@@ -0,0 +1,279 @@
+from collections import defaultdict, Counter
+import networkx as nx
+import pysmiles
+from pysmiles.write_smiles import _get_ring_marker
+
+def format_node(molecule, current):
+    node = "[#{}]".format(molecule.nodes[current]['fragname'])
+    return node
+
+def write_cgsmiles_res_graph(molecule):
```

Comment:
How does this function differ from the pysmiles one?

Reply:
very similar; the two differences are the formatting of the nodes and that you cannot remove explicit hydrogen atoms in this case. However, this would require pysmiles API to change and not sure that is desirable